src/main/java/io/vertx/core/streams/impl/InboundBuffer.java

```diff
@@ -243,6 +246,7 @@ private void handleException(Throwable err) {
     Handler<Throwable> handler;
     synchronized (this) {
       if ((handler = exceptionHandler) == null) {
+        log.error("Unhandled exception", err);
```

Comment:
I prefer to not have this here and instead let the `InboundBuffer` user handle logging like you did for `NetSocket`

Reply:
Yep, I have added logging to `InboundBuffer` as well in my previous commits, but I think it's a good practice to at least log the error when there's no explicit handler set on each level.

Actually, I would even re-throw the exception even further when there's no handler as it will help to detect those kind of errors earlier, but , I assume, that wouldn't be really backward compatible at the moment.

Reply:
please remove the logging from there, the class using it should be responsible handling logging.

Reply:
push a commit that drops logging for unhandled exceptions.